debug_toolbar/static/debug_toolbar/css/toolbar.css

```
@@ -607,6 +607,18 @@
 #djDebug .djdt-highlighted {
     background-color: lightgrey;
 }
+@keyframes flash-new {
+    from {
+        background-color: green;
+    }
+    to {
+        background-color: inherit;
+    }
+}
+#djDebug .flash-new {
+    animation: flash-new 1s;
+}
+
```

I think it would be a good idea to namespace this.

```suggestion
@keyframes djdt-flash-new {
    from {
        background-color: green;
    }
    to {
        background-color: inherit;
    }
}
#djDebug .flash-new {
    animation: djdt-flash-new 1s;
}

```